build.gradle.kts

```diff
@@ -133,8 +132,8 @@ if (HostManager.hostIsMac) {
     val testIosSim by tasks.registering(Exec::class) {
         group = "verification"
         dependsOn(linkTestDebugExecutableIosSim)
-        executable("xcrun")
```

Comment:
Can we leave this? I generally prefer separation of executable and args

Reply:
Doesn’t this show you an error in IntelliJ the whole time too?

Reply:
OK, seems like `setArgs()` removes the error reported by IntelliJ.